### PR TITLE
bugfix | tunning config

### PR DESCRIPTION
**Pull request by WoLkErSs**
[Replace this text by the issue key](replace_this_text_by_the_issue_link)

### Description

Bump ruby to 2.7.2
Tunned dockerfile
Added redirect to admin dashbord
Have united `Macros::ModelDestroy` and `Macros::ModelDelete` to `Macros::ModelRemove`
Added `sentry_dsn` to staging and production credentials

### Before submitting the merge request make sure the following are checked

- [x] Followed the style guidelines of this project
- [x] Performed a self-review of own code
- [x] Wrote the tests that prove that fix is effective/that feature works

### CODE REVIEW

File: Dockerfile [hunk 1]
```diff
@@ -1,31 +1,37 @@
...
+USER root
+
+RUN yarn install --check-files
 
 COPY . .
 
-FROM ruby:2.6.5-alpine
+RUN RAILS_ENV=development rake assets:precompile
+
+RUN rm -rf node_modules spec tmp/cache
```
Comment (dzemlianoi-double) [inline Dockerfile:32]: Do we need to delete node modules at development env?

Comment (dzemlianoi-double) [reply Dockerfile:32]: And the specs? Could we run specs under docker?

File: Dockerfile [hunk 2]
```diff
@@ -43,5 +49,3 @@ COPY --from=Builder --chown=1000:1000 $APP_HOME $APP_HOME
 WORKDIR $APP_HOME
 
 USER $APP_USER
-
-CMD bundle exec puma -C config/puma.rb
```
Comment (dzemlianoi-double) [inline Dockerfile:47]: So we don't need to run puma at all or we do it in compose?

File: app/concepts/api/v1/users/reset_passwords/operation/create.rb [hunk 3]
```diff
@@ -10,7 +10,8 @@ class Create < ApplicationOperation
     step Macro::Contract::Schema(Api::V1::Users::ResetPasswords::Contract::Create)
     step Contract::Validate(), fail_fast: true
     step Model(Account, :find_by_email, :email)
-    fail Macro::Semantic(failure: :not_found), fail_fast: true
+    fail Macro::Semantic(failure: :not_found)
+    fail Macro::AddContractError(email: 'errors.session.email_not_found'), fail_fast: true
```
Comment (dzemlianoi-double) [inline app/concepts/api/v1/users/reset_passwords/operation/create.rb]: IT'S NOT SESSION! IT'S RESET_PASSWORD!!!!!

File: config/locales/en.errors.yml [hunk 4]
```diff
@@ -13,6 +13,8 @@ en:
       user_password:
         eql?: should match to password
     session:
+      email_not_found: Email not found
```
Comment (dzemlianoi-double) [inline config/locales/en.errors.yml]: move to reset password

**Review by dzemlianoi-double — CHANGES_REQUESTED** [review]

File: Dockerfile (hunk 2 above)
Comment (WoLkErSs) [reply Dockerfile:47]: It runs in `docker-compose` and `container_definitions` for staging/production environment

File: config/credentials/credentials.yml [hunk 5]
```diff
@@ -0,0 +1,25 @@
...
+
+db:
+  host: Domain of RDS instance
+  port: 5432
+  database: Database name from RDS
+  user: User name for access to DB on RDS
+  password: Password to RDS DB
+  pool: 5
+
+sentry_dsn: Not used. Used for notify in a SLACK channel.
```
Comment (dzemlianoi-double) [inline config/credentials/credentials.yml:25]: It's not a file - it's a shame

**Review by dzemlianoi-double — APPROVED** [review]

**Pull request merged**